.gitignore

```diff
@@ -38,3 +38,4 @@ shared-local-instance.db
 dataactcore/scripts/dropMany.py
 runDynamo.py
 dataactvalidator/config/sf_133.csv
+runTests.py
```

Comment:
That's a different runTests, I was trying to have a script that would send the pytest shell command to run both the integration and unit tests, and I assumed we didn't want that in the repo.  I'll rename this to something less confusing.